ClientApp/src/pages/Blog.js

```diff
@@ -0,0 +1,82 @@
+import React, {useState} from 'react'
+import {useSelector} from 'react-redux'
+import {useHistory} from 'react-router-dom'
+import {Col, Row} from 'reactstrap'
+
+import '../styles/components/blog.scss'
+import Layout from "../components/Layout";
+
+const Blog = props => {
+    const history = useHistory();
+    const blogs = useSelector(state => state.blogs.all);
+    const blog = blogs.find(temp => parseInt(temp.id, 10) === parseInt(props.match.params.id))
```

Comment:
This works but we plan on adding pagination. This wouldn't work if a person selected a blog on the second page then refreshed as the initial query will always pull from the first page from the server. You should set up a useEffect function that checks to see if this returns anything, if not, you need to make a async request to the server the get details about this blog

Reply:
I think making a request to the server, no checks, is the way to go then